string credential
---
int16 status
string message
